assets/src/js/Components/ArchivePicker.js

```diff
@@ -0,0 +1,151 @@
+const __ = ( str ) => wp.i18n.__( str, 'planet4-master-theme-backend' );
```

Comment:
The Loco translate plugin string extractor not able to evaluate variables/constants. 
We need to use - `__('string to translate', 'text-domain')` format of translation function.
Even we are not able to use textdomain variable here.

Reply:
Right, good catch, didn't think of that :+1: Too bad, I thought I found a way to only specify the domain once per file, but probably would be hard to make this work with Loco, so I'll remove it.